AzureStore.UI/app/src/components/Basket/Basket.tsx

```diff
@@ -96,20 +106,23 @@ export default class Basket extends React.Component<Props> {
       this.props.submitOrder(order);
     }
 
-    getTotal(items: any): number {
+    getTotal(items: OrderItem[]): number {
       let total = 0;
-      items.forEach((item: any) => total+=item.product.amount*item.quantity);
+      items.forEach((item: any) => total+=item.product.price*item.quantity);
       return total;
     }
 
     render() {
-      const tableStyle = {
-        margin: "40px auto",
-        width: "800px"
-      };
-
       return (
-        <Container style={{padding: "40px"}}>
+        <Container style={{padding: "40px 20px"}}>
+          <StyledBreadcrumbs aria-label="breadcrumb">
+                <StyledLink color="inherit" to="/">
```

Comment:
Maybe we need to add 'inherit' color by default for StyledLink?

Reply:
it's already there, I will just remove it